### PR TITLE
fix: use object rather than label in top n selection

### PR DESCRIPTION
**Pull request by aaron-steinfeld**

### CODE REVIEW

File: projects/observability/src/shared/components/top-n/top-n-chart.component.ts [hunk 1]
```diff
@@ -63,20 +63,20 @@ export class TopNChartComponent implements OnChanges {
...
 }
 export interface TopNData {
   label: string;
   value: number;
 }
 
-interface TopNItemOptions extends TopNData {
+interface TopNItemOption<T extends TopNData> extends TopNData {
   width: string;
+  original: T;
```
Comment (anandtiwary) [inline projects/observability/src/shared/components/top-n/top-n-chart.component.ts:81]: What's the reason for this change?

Comment (aaron-steinfeld) [reply projects/observability/src/shared/components/top-n/top-n-chart.component.ts:81]: The previous API had no way of disambiguating two objects with identical labels.

Comment (anandtiwary) [reply projects/observability/src/shared/components/top-n/top-n-chart.component.ts:81]: I see. Thanks!
